### ISSUE TITLE
prefilling check boxes and radio not working

### ISSUE THREAD

**Issue by mahiagit** [pdffillerjs/pdffiller#61]
I am providing the existed data to the Pdf form. All the text data is being nicely populated with the pdf filler but when going for the checkboxes or radio nothing is working out. Creating the data as below:

data = {
            "NAME":"Test",
            "Test CheckBox1":"Yes",
            "Test CheckBox2":true,
            "Test CheckBox3"":1,
            "Test CheckBox4 ":"On"
    } 

Not any of the provided options/value for the checkbox is working . Kindly suggest the checkbox or radio is being shown as selected by providing prefilled data

**Comment by maximilianschmidt** [comment]
have you checked if the Export-Value of the checkbox is indeed one of the values you set? it might differ. check with adobe acrobat -> right click on the checkbox -> properties -> options -> export value.

**Comment by mahiagit** [comment]
Yes there was difference in the values. thanks @maximilianschmidt
